internal/http_api/api_response.go

```diff
@@ -43,11 +42,9 @@ func PlainText(f APIHandler) APIHandler {
 		}
 		switch d := data.(type) {
 		case string:
-			w.Header().Set("Content-Length", strconv.Itoa(len(d)))
```

Comment:
`net/http` will actually set a content-length for us in most cases (depending on how content is written out). This code here became useless because of the compression middleware.